docs/blog/2018-2-7-jam-out-your-blog/index.md

```diff
@@ -0,0 +1,158 @@
+
+I’m willing to assume that most readers have been here before and are currently cringing in unison. Luckily, I found a solution. 
+
+The goal of the following article is to share a tool set that I found to work effectively at trying to bring order to the chaos. My goal is to provide a workflow that will be ideal for personal blogs and hand-off projects where some sort of a CMS is within the project's scope. 
+
+As a preface, I will be referencing a wonderful article written by Josh Weaver, [Gatsby + Contentful + Netlify (and Algolia)](https://www.gatsbyjs.org/blog/2017-12-06-gatsby-plus-contentful-plus-netlify/). Josh’s article spoke to my goals and frustrations perfectly and inspired me to share this alternative approach. 
+
+_**Let’s get into it.**_
+
+<div style="width:100%;height:0;padding-bottom:55%;position:relative;"><iframe src="https://giphy.com/embed/WDtCGLCXBAgow" width="100%" height="100%" style="position:absolute" frameBorder="0" class="giphy-embed" allowFullScreen></iframe></div><p><a href="https://giphy.com/gifs/tom-cruise-top-gun-WDtCGLCXBAgow">via GIPHY</a></p>
+
+## My Requirements
+
+### Like Josh’s requirements, my list includes:
+
+*   Speed - this includes development, website performance, and deployment
+*   Development Experience - I want to use React and Webpack 
+*   Maintainability - ease with which anyone can create, update and delete posts
+*   Scalability - need to make sure that performance is not hindered by a growing needs
+*   Lean - simplify the tools and services use to a bare minimum
+*   Revision History - for accountability and allowing for versioning 
+*   Inexpensive Hosting - again, to maximize value and keep personal costs down
+*   NO WordPress - similarly, I don’t want to deal with updates, themes, hacks...etc
+*   Continuous Deployment - the site needs to always be up and running and current
+
+For the sake of brevity, let's infer that speed, development experience, scalability and lack of WordPress will be handled by Gatsby and the JAMstack. If you’re unfamiliar, take a look at [http://jamstack.org/. ](http://jamstack.org/)
+
+## CMS-Induced Headaches
+
+From a non-technical content-creator perspective, more UI and less code is ideal. So if we want to avoid WordPress, how can we provide the experience of  a simple login where a user can create, edit and publish posts?
+
+Josh’s article provides a solution by way of using Contentful to manage content on his company’s documentation website. 
+
+### Here are some Pros (at the time of writing):
+
+*   Clean UI
+*   Free Tier
+*   User Roles (admins, editors, content authors)
+*   Comprehensive tutorials
+*   Fully-customizable front end
+*   Article search and filters
+
+That seems like a pretty solid win for Contentful, right? Well, there are a few issues. 
```

Comment:
Can you tone the language down here a bit? This would be going out as an official blog post and this isn't the right tone. Contentful + Gatsby is a great solution for a lot of people and we shouldn't throw shade on the experience. It's fine if you highlight the issue you had with it and why you ultimately went with a different solution.

Reply:
Absolutely!